core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -255,13 +255,7 @@ public BigQuerySqlDialect(SqlDialect.Context context) {
     if (interval.getSign() == -1) {
       writer.print("-");
     }
-    Long intervalValueInLong;
-    try {
-      intervalValueInLong = Long.parseLong(literal.getValue().toString());
-    } catch (NumberFormatException e) {
-      throw new RuntimeException("Only INT64 is supported as the interval value for BigQuery.");
-    }
-    writer.literal(intervalValueInLong.toString());
+    writer.literal(literal.getValue().toString());
```

Comment:
Hm, this shouldn't be in the diff. Doesn't this change already exist in master?

Reply:
No idea why any of what you pointed out is happening :/ Maybe because this isn't the original branch?

Reply:
Actually the same thing is happening with parsing_exodus into master: https://github.com/googleinterns/calcite/compare/master...parsing_exodus

Reply:
My guess is that since the /intermediate/ directory didn't exist before git considers it as a new file.

Reply:
Interestingly, not all .ftl files are showing this issue. For example, this file looks like it was moved/renamed:

parsing/dialects/defaultdialect/parserImpls.ftl → parsing/intermediate/dialects/defaultdialect/parserImpls.ftl

Reply:
I don't believe that I made any changes to defaultdialect which is maybe why it is showing up properly.

Reply:
I don't think it's just a matter of which files were edited because dialect1/parserImpls.ftl also was moved/renamed properly:
parsing/dialects/dialect1/parserImpls.ftl → parsing/intermediate/dialects/dialect1/parserImpls.ftl

Can you try doing a "git mv" on the files that are showing up as deleted/created? (https://git-scm.com/docs/git-mv)

Reply:
Sorry if I'm missing something, but how can I do this if the changes have already been made?

Reply:
You should be able to use some combination of "git reset" and "git checkout" with a specific filename to undo the changes in that file to match the version in master (https://git-scm.com/docs/git-checkout).

Just make sure to copy/paste the contents of the file into some place safe so that you can copy/paste them back in after you do the "git mv".

Reply:
I ran through the steps I recommended to you and I'm still seeing GitHub show these files as being deleted/added rather than modified/renamed, even though git was saying otherwise. I think we can chalk this up to a GitHub diff visualization bug.